miniapp/miniapp_cholesky.cpp

```diff
@@ -392,13 +394,16 @@ options_t check_options(hpx::program_options::variables_map& vm) {
       vm["matrix-size"].as<SizeType>(), vm["block-size"].as<SizeType>(),
       vm["grid-rows"].as<int>(),        vm["grid-cols"].as<int>(),
 
-      vm["nruns"].as<int64_t>(),        CHECK_RESULT::NONE,
+      vm["nruns"].as<int64_t>(),        vm["nwarmups"].as<int64_t>(),
+      CHECK_RESULT::NONE,
   };
 
   DLAF_ASSERT(opts.m > 0, "matrix size must be a positive number!", opts.m);
   DLAF_ASSERT(opts.mb > 0, "block size must be a positive number!", opts.mb);
   DLAF_ASSERT(opts.grid_rows > 0, "number of grid rows must be a positive number!", opts.grid_rows);
   DLAF_ASSERT(opts.grid_cols > 0, "number of grid columns must be a positive number!", opts.grid_cols);
+  DLAF_ASSERT(opts.nruns > 0, "number of runs must be a positive number!", opts.nruns);
+  DLAF_ASSERT(opts.nwarmups >= 0, "number of warmup runs must be a nonnegative number!", opts.nwarmups);
```

Comment:
I suppose the message will not stay for too long. Would you like me to leave removing it to #295 or get rid of it right away?

Reply:
Yes, just go ahead and remove it, it's redundant.

Reply:
Done.